src/tools/elektrad/temp-elektra.pc.in

```diff
@@ -1,10 +1,10 @@
-prefix=@CMAKE_BINARY_DIR@
-libdir=${prefix}/lib
-srcinclude=${prefix}/src/include
-buildinclude=@CMAKE_SOURCE_DIR@/src/include
+prefix="@CMAKE_BINARY_DIR@"
+libdir="${prefix}/lib"
+srcinclude="${prefix}/src/include"
+buildinclude="@CMAKE_SOURCE_DIR@/src/include"
 
 Name: @PACKAGE_NAME@
 Description: @PACKAGE_DESCRIPTION@
 Version: @KDB_VERSION@
-Cflags: -I${srcinclude} -I${buildinclude}
-Libs: -L${libdir} -lelektra-kdb -lelektra-core
+Cflags: -I"\"${srcinclude}\"" -I"\"${buildinclude}\""
+Libs: -L"\"${libdir}\"" -lelektra-kdb -lelektra-core
```

Comment:
```suggestion
Cflags: -I"${srcinclude}" -I"${buildinclude}"
Libs: -L"${libdir}" -lelektra-kdb -lelektra-core
```

For my tests this is enough. Like in a shell you only need to put quotes around the `${variable}`s. If I do do this with this test file locally:

```
prefix=test string abc
exec_prefix="${prefix}"/bin
libdir="${prefix}"/lib
includedir="${prefix}"/include/elektra
plugindir="${prefix}"/lib/elektra
tool_execdir="${prefix}"/lib/elektra/tool_exec
templatedir="${prefix}"/share/elektra/templates
build_type=shared

Name: 
Description: 
Version: 0.9.14
Libs: -L"${libdir}" -lelektra-kdb -lelektra-core
Cflags: -I"${prefix}"/include
```

And run `PKG_CONFIG_PATH=path/to/folder/of/file/above pkg-config --libs --cflags elektra` I get:

```
-Itest\ string\ abc/include -Ltest\ string\ abc/lib -lelektra-kdb -lelektra-core 
```

AFAICT this should be interpreted correctly by `cgo`. If that is not the case, this file is not the problem.

Reply:
Yes, it worked locally as well. I had it this way, but jenkins was not happy again.